backend/controllers/usersControllers.js

```diff
@@ -66,6 +68,63 @@ module.exports.register = async (req, res) => {
   };
 
 
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
```

Comment:
Create a variable inside the .**env** file and store the **SMTP** host value there.